test/components/nested-vertical-list.html

```diff
@@ -0,0 +1,935 @@
+<!DOCTYPE html>
+<html lang="">
+
+<head class="">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+	<link rel="stylesheet" href="../../library/css/index.css">
+	<title></title>
+</head>
+
+<body class="light">
+    <div class="ui twenty-eighty-container faculty-vertical-list">
+        
+        <div class="first first-vertical-list">
+
+            <div class="vertical-list">
+                <div class="vertical-list-head ui sub-heading" id="1" onclick="head_click(this.id)">Biosketch</div>
+                <div class="vertical-list-nest">
+                    <div class="vertical-list-nest-content ui one-liner-captions" id="edudetails" onclick="content_click(this.id)">Educational Details</div>
+                    <div class="vertical-list-nest-content ui one-liner-captions" id="profbg" onclick="content_click(this.id)">Professional Background</div>
+                    <div class="vertical-list-nest-content ui one-liner-captions" id="adminpos" onclick="content_click(this.id)">Administrative Positions</div>
```

Comment:
There is no need of a vertical-list-line class, just target the ```hr``` tag as a children of vertical-list